tests/validation/test_cat_vrs_models.py

```diff
@@ -0,0 +1,350 @@
+"""Test Cat VRS Pydantic models"""
+
+from copy import deepcopy
+
+import pytest
+from ga4gh.cat_vrs import models, recipes
+from ga4gh.core.models import (
+    MappableConcept,
+    code,
+)
+from ga4gh.vrs.models import CopyChange
+
+
+def def_allele_constr_empty_relations(
+    is_empty_list=True,
+) -> models.DefiningAlleleConstraint:
+    """Return DefiningAlleleConstraint with either empty relations or null relations
+
+    :param is_empty_list: ``True`` if relations should be empty list. Otherwise,
+        relations will be null
+    """
+    return models.DefiningAlleleConstraint(
+        relations=[] if is_empty_list else None, allele="allele.json#/1"
+    )
+
+
+@pytest.fixture(scope="module")
+def copy_change_constr():
+    """Create test fixture for copy change constraint"""
+    return models.CopyChangeConstraint(copyChange=CopyChange.EFO_0030069.value)
+
+
+@pytest.fixture(scope="module")
+def members():
+    """Create test fixture for members"""
+    return {"members": ["variation.json#/1"]}
+
+
+@pytest.fixture(scope="module")
+def defining_loc_constr():
+    """Create test fixture for defining location constraint"""
+    return models.DefiningLocationConstraint(
+        relations=[
+            MappableConcept(primaryCode=code(models.Relation.LIFTOVER_TO.value))
+        ],
+        location="location.json#/1",
+        matchCharacteristic=MappableConcept(label="test"),
+    )
+
+
+def test_copy_change_constraint():
+    """Test the CopyChangeConstraint validator"""
+    # Valid Copy Change
+    assert models.CopyChangeConstraint(copyChange=CopyChange.EFO_0030069.value)
+
+    # Invalid Copy Change
+    with pytest.raises(ValueError, match="copyChange, 0030069, not one of"):
+        models.CopyChangeConstraint(copyChange="0030069")
+
+
+def test_protein_sequence_consequence(defining_loc_constr, members):
+    """Test the ProteinSequenceConsequence validator"""
+    # Valid PSC
+    valid_params = deepcopy(members)
+    valid_params["constraints"] = [
+        models.Constraint(
+            root=models.DefiningAlleleConstraint(
+                relations=[
+                    MappableConcept(
+                        primaryCode=code(models.Relation.TRANSLATES_FROM.value)
+                    )
+                ],
+                allele="allele.json#/1",
```

Comment:
Does this refer to [Allele.json](https://github.com/ga4gh/vrs/blob/0106b48c1db0b420efd787cc769e4b35f415068e/schema/vrs/json/Allele) in vrs' schemas? If so, should this be capitalized?

Reply:
Oh, this was just a dummy IRI. I can make a comment